doc/contribute.md

```diff
@@ -506,8 +456,6 @@ Shared credentials are stored in a 1Password vault. Get in touch with one of the
 
 The HoloViz project maintains a blog at https://blog.holoviz.org/ where new major releases are announced.
 
-The former PyViz-named blog is still alive at http://blog.pyviz.org/.
```

Comment:
Just for reference, I was documenting that sort of thing here because there's a real connection between HoloViz and PyViz (https://github.com/pyviz/pyviz.org/issues/190). Now I opened this issue I think it's okay to remove this comment.

But yes, in general we should consider this document our knowledge base and not hesitate to add information even if they don't appear so relevant (it's better here than in someone's head only!)

Reply:
OK. The comment is no longer there though.